extension/contentScript.js

```diff
@@ -11,122 +11,108 @@
 // @grant        none
 // ==/UserScript==
 
-(function() {
-    'use strict';
+'use strict';
 
-    const isWatchVideoPage = () => window.location.pathname == "/watch"
+const isWatchVideoPage = () => window.location.pathname === "/watch"
 
-    const fluff = ['#secondary-inner', '#info', '#meta', '#comments', '#masthead-container', '#speedyg']
+const fluff = ['#secondary-inner', '#info', '#meta', '#comments', '#masthead-container', '#speedyg']
 
-    const setFluffDisplay = (value) => {
-        fluff.forEach(selector => {
-            document.querySelectorAll(selector).forEach(el => {
-                el.style.display = value
-            })
+const setFluffDisplay = (value) => {
+    fluff.forEach(selector => {
+        document.querySelectorAll(selector).forEach(el => {
+            el.style.display = value
         })
-    }
-
-    const hideFluff = () => {
-        console.log(`[YT-FFT] Hiding fluff`)
-        setFluffDisplay('none')
-    }
-    const showFluff = () => {
-        console.log(`[YT-FFT] Showing fluff again`)
-        setFluffDisplay('')
-    }
-
-    const isInFullscreen = () => !!document.fullscreenElement
+    })
+}
 
-    // Opposite since we will be checking this only after it's changed by YouTube
-    const checkIfDirectionIsLeavingFullscreen = () => !isInFullscreen()
+const hideFluff = () => {
+    setFluffDisplay('none')
+}
+const showFluff = () => {
+    setFluffDisplay('')
+}
 
-    const playerWidthDiff = (player) => Math.abs(player.clientWidth - window.innerWidth)
-    const playerHeightDiff = (player) => Math.abs(player.clientHeight - window.innerHeight)
+const isInFullscreen = () => Boolean(document.fullscreenElement)
 
-    const playerIsFullWidth = (player) => playerWidthDiff(player) <= 2
-    const playerIsFullHeight = (player) => playerHeightDiff(player) <= 2
+// Opposite since we will be checking this only after it's changed by YouTube
+const checkIfDirectionIsLeavingFullscreen = () => !isInFullscreen()
 
-    const debugPlayerSize = (player) => {
-        console.log('[YT-FFT] w:', player.clientHeight, window.innerHeight, 'h:', player.clientWidth, window.innerWidth)
-    }
+const playerWidthDiff = (player) => Math.abs(player.clientWidth - window.innerWidth)
+const playerHeightDiff = (player) => Math.abs(player.clientHeight - window.innerHeight)
 
-    const playerIsFullscreen = () => {
-        const player = document.querySelector('ytd-player')
-        debugPlayerSize(player)
```

Comment:
Without any explanation as to how performance is improved, I assume you mean it's by not having to log the player size debugging? I never noticed an impact. And it'd be helpful information to supply when reporting an issue, so I'd like to leave it in. If you're concerned about it, a more useful change would be to make it toggleable via config.

Reply:
`console.log` is useful when you are actively debugging the code. I don't think it is useful to print information and fill the dev console from these messages.